### ISSUE TITLE
Mobile Menu Close Icon Position

### ISSUE THREAD

**Issue by kleofont** [soflyy/oxygen-bugs-and-features#1963]
**Describe the feature you'd like to see included in Oxygen.**
Please move the default close mobile menu icon position to right. Right now it is set to left and we have to offset it to the left
which created more problems on varois devices than it helps. Most people are right handed so it is just easier for them to tap
the close icon when it's on the right

**What are the use cases for this feature?**
Any responsive website

**Examples of this feature or functionality.**
Basically any site builder out there
